server/index.js

```diff
@@ -1,11 +1,30 @@
-if (process.env !== 'production') {
+if (process.env.NODE_ENV !== 'production') {
   require('dotenv').config();
 };
+
+
 const bodyParser = require('body-parser');
-const express = require('express');
+const express = require('express'),
+  app = express(),
+  session = require('express-session');
+const MongoStore = require('connect-mongo')(session);
+
+const db = require('./models/db');
+const user = require('./models/users');
+const question = require('./models/questions');
+const answer = require('./models/answers');
 
 const PORT = process.env.PORT || 3001;
-const app = express();
+
+// create a session with a session ID
+app.use(session({
+  secret: process.env.SECRET,
+  resave: true,
+  saveUninitialized: false,
+  store: new MongoStore({
+  mongooseConnection: db
```

Comment:
this could go on one line. If you want to break it up that's fine, but `mongooseConnection` should be indented

Reply:
Readability is not good when on one line. Fixed indentation.